[rospy.client][INFO] 2021-05-09 10:50:12,029: init_node, name[/plot_characteristic_curve_node], pid[2743]
[xmlrpc][INFO] 2021-05-09 10:50:12,029: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-05-09 10:50:12,030: Started XML-RPC server [http://shang:45887/]
[rospy.init][INFO] 2021-05-09 10:50:12,030: ROS Slave URI: [http://shang:45887/]
[rospy.impl.masterslave][INFO] 2021-05-09 10:50:12,030: _ready: http://shang:45887/
[rospy.registration][INFO] 2021-05-09 10:50:12,031: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-05-09 10:50:12,031: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-05-09 10:50:12,131: registered with master
[rospy.rosout][INFO] 2021-05-09 10:50:12,131: initializing /rosout core topic
[rospy.rosout][INFO] 2021-05-09 10:50:12,133: connected to core topic /rosout
[rospy.simtime][INFO] 2021-05-09 10:50:12,134: initializing /clock core topic
[rospy.simtime][INFO] 2021-05-09 10:50:12,137: connected to core topic /clock
[rospy.internal][INFO] 2021-05-09 10:50:12,308: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2021-05-09 10:50:12,571: topic[/clock] adding connection to [http://shang:43991/], count 0
[rospy.internal][WARNING] 2021-05-09 10:53:10,469: Unknown error initiating TCP/IP socket to shang:48267 (http://shang:43991/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 651, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2021-05-09 10:53:10,469: topic[/clock] removing connection to http://shang:43991/
[rospy.core][INFO] 2021-05-09 10:58:20,107: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-05-09 10:58:20,127: topic[/rosout] removing connection to /rosout
[rospy.impl.masterslave][INFO] 2021-05-09 10:58:20,128: signal-2
[rospy.core][INFO] 2021-05-09 10:58:20,744: signal_shutdown [atexit]
[rospy.client][INFO] 2021-05-09 10:58:24,263: init_node, name[/plot_characteristic_curve_node], pid[5270]
[xmlrpc][INFO] 2021-05-09 10:58:24,263: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-05-09 10:58:24,264: Started XML-RPC server [http://shang:34385/]
[rospy.impl.masterslave][INFO] 2021-05-09 10:58:24,264: _ready: http://shang:34385/
[rospy.init][INFO] 2021-05-09 10:58:24,264: ROS Slave URI: [http://shang:34385/]
[xmlrpc][INFO] 2021-05-09 10:58:24,265: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2021-05-09 10:58:24,269: Registering with master node http://localhost:11311
[rospy.init][INFO] 2021-05-09 10:58:24,365: registered with master
[rospy.rosout][INFO] 2021-05-09 10:58:24,365: initializing /rosout core topic
[rospy.rosout][INFO] 2021-05-09 10:58:24,368: connected to core topic /rosout
[rospy.simtime][INFO] 2021-05-09 10:58:24,370: initializing /clock core topic
[rospy.simtime][INFO] 2021-05-09 10:58:24,374: connected to core topic /clock
[rospy.internal][INFO] 2021-05-09 10:58:24,530: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2021-05-09 10:58:24,871: topic[/clock] adding connection to [http://shang:36863/], count 0
[rospy.internal][WARNING] 2021-05-09 11:01:09,595: Unknown error initiating TCP/IP socket to shang:36501 (http://shang:36863/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 651, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2021-05-09 11:01:09,595: topic[/clock] removing connection to http://shang:36863/
[rospy.core][INFO] 2021-05-09 11:01:19,531: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-05-09 11:01:19,548: topic[/rosout] removing connection to /rosout
[rospy.impl.masterslave][INFO] 2021-05-09 11:01:19,549: signal-2
[rospy.core][INFO] 2021-05-09 11:01:20,172: signal_shutdown [atexit]
